source/code-snippets/collation/aggregation-collation.js

```diff
@@ -0,0 +1,33 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority&useUnifiedTopology=true";
+
+// Create a new MongoClient
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    // Connect the client to the server
+    await client.connect();
+
+    const db = client.db("myproject");
+    const collection = db.collection("names");
+
+    const cursor = collection.aggregate(
+      [
+        { $group: { "_id": "$first_name", "nameCount": { "$sum": 1 } } },
+        { $sort: { "_id": 1 } },
+      ],
+      { collation: { locale: "de@collation=phonebook" } },
+    );
+
+    console.log(await cursor.toArray());
+  } finally {
+    // Ensures that the client will close when you finish/error
+    await client.close();
+  }
+}
```

Comment:
I feel like it's hard to read these examples because they use an entire runnable file, which distracts from the lines that are actually important. Can we trim down the included lines using `:startsafter:` and `:endsbefore:` like I did for the CRUD examples? I'm thinking that we really just need the `collection.<method>` code to get the point across - users can always consult the usage examples for full runnable files.

Reply:
I'm removing the calls and any additional code that makes it seem like it's being called inside an async function. I'll update the descriptions too since the database and collection names are not necessary.